cdap-cli/src/main/java/co/cask/cdap/cli/util/table/AltStyleTableRenderer.java

```diff
@@ -69,7 +69,7 @@
  */
 public class AltStyleTableRenderer implements TableRenderer {
 
-  private static final int DEFAULT_WIDTH = 80;
+  public static final int DEFAULT_WIDTH = 80;
```

Comment:
I opened another PR for making line width a property of `CLIConfig`, which addresses this. https://github.com/caskdata/cdap/pull/1656